test/dotcom_web/views/cms_view_test.exs

```diff
@@ -135,7 +135,7 @@ defmodule DotcomWeb.CMSViewTest do
         )
 
       # could also be November 6th, 1:00 AM (test daylight savings)
-      expected = "November 5, 2016 1 AM - November 6, 2016 2 AM"
+      expected = "November 5, 2016 1 AM - November 6, 2016 1 AM"
```

Comment:
📓 Ambiguous daylight savings times were wrong in previous versions of Timex. We probably shouldn't test this in our code because it just tests the underlying library. But, they're already there so we can just correct them.